src/applications/hca/config/chapters/veteranInformation/birthInformation.js

```diff
@@ -32,8 +30,11 @@ export default {
           cityOfBirth,
           stateOfBirth: {
             type: 'string',
-            enum: [...states50AndDC.map(object => object.value), 'Other'],
-            enumNames: [...states50AndDC.map(object => object.label), 'Other'],
+            enum: [...STATES_50_AND_DC.map(object => object.value), 'Other'],
+            enumNames: [
+              ...STATES_50_AND_DC.map(object => object.label),
+              'Other',
+            ],
```

Comment:
These should really be moved to the schema file itself. No reason to be doing these here.